### DISCUSSION TITLE
How to add columns?

### DISCUSSION THREAD

**Discussion by Franklin098** [react-page/react-page#892] [Q&A]
I'm following the get started guide, and I want to know how can I add columns, because at this moment I can only add text, images, and videos, in rows.

**Comment by macrozone** [comment]
hi @Franklin098 you can drag elements around so you can move them to the left or right of existing elements and you can resize the resulting columns. There are no explicit "columns" that you can add, but maybe i do not understand the question correctly?

by the way be sure to use the new beta version as it will be released soon (in the next 2 weeks).

https://react-page.github.io/beta

(install it using the the beta tag: `yarn add @react-page/editor@beta` or `npm install --save @react-page/editor@beta`)

**Comment by zehawki** [comment]
And I want to further add that the current mechanism is singularly powerful. You are not constrained to pre-selecting a certain layout of columns etc. Its very fluid and I've not seen anything as good as this. As @macrozone said. just move elements around and columns will be created, resize them as you see fit etc. And they will be automatically responsive as well,

**Comment by macrozone** [comment]
i also want to improve on the resizing by providing an alterntive to dragging: https://github.com/react-page/react-page/issues/898
